controllers/create_helper.go

```diff
@@ -316,6 +330,11 @@ func (b *RestoreHelper) phase(phase veleroapi.RestorePhase) *RestoreHelper {
 	return b
 }
 
+func (b *RestoreHelper) setDeleteTimestamp(deletionTimestamp metav1.Time) *RestoreHelper {
```

Comment:
Is this used anywhere?

Reply:
yes, is used in a testcase, lint would error otherwise